apis/pkg/v1alpha1/revision_types.go

```diff
@@ -73,8 +71,4 @@ type PackageRevisionStatus struct {
 
 	// References to objects owned by PackageRevision.
 	ObjectRefs []runtimev1alpha1.TypedReference `json:"objectRefs,omitempty"`
-
-	// PermissionRequests are additional permissions that should be added to a
-	// packaged controller's service account.
-	PermissionRequests []rbac.PolicyRule `json:"permissionRequests,omitempty"`
```

Comment:
Since we are taking the approach of removing anything that isn't currently used, do you think we should keep `Crossplane` and `DependsOn` here? I suspect they will likely both be supported in next release (v0.14) for context

Reply:
If they won't be functional in v0.13 I think we should remove them. I'd prefer that to be done in a separate PR, since neither of them relate to the RBAC manager.